waddrmgr/address.go

```diff
@@ -98,6 +101,7 @@ type managedAddress struct {
 	internal         bool
 	compressed       bool
 	used             bool
+	watchingOnly     bool
 	pubKey           *btcec.PublicKey
 	privKeyEncrypted []byte
```

Comment:
OK, I'll try to replace it with nilled privKeyEncrypted. Also, as per the IRC discussion, I think if we really want to store `watchingOnly` bool, it could be a part of the flag field.